proposals/collection-expression-arguments.md

```diff
@@ -303,10 +304,9 @@ l = [with(default)];           // error: ambiguous constructor
 ```
 
 If the target type is a type with a *create method*, then:
-* The *argument list* is the `with()` *argument list* followed by a *collection expression* containing the elements only (no arguments).
-* [*Overload resolution*](https://github.com/dotnet/csharpstandard/blob/standard-v7/standard/expressions.md#1264-overload-resolution) is used to determine the best factory method from the *argument list* from the [*applicable create methods*](#applicable-create-method):
-* If a best factory method is found, the method is invoked with the *argument list*.
-  * If the factory method has a `params` parameter, the invocation may be in expanded form.
+* [*Overload resolution*](https://github.com/dotnet/csharpstandard/blob/standard-v7/standard/expressions.md#1264-overload-resolution) is used to determine the best factory method from the candidates.
+* The set of candidate factory methods is the [*create method candidates*](#create-method-candidates) for the target type that are applicable with respect to the *argument list* as defined in [*applicable create methods*](#applicable-create-method).
+* If a best factory method is found, the method is invoked with the *argument list* appended with a `ReadOnlySpan<T>` containing the elements.
```

Comment:
not quite understanding what this means.  i was expecting something about how the `with(...)` arguments need to be the first `N-1` args of the final create method.

Reply:
ah, gotcha.